src/DynamoCore/Graph/Nodes/ZeroTouch/DSFunctionBase.cs

```diff
@@ -76,6 +77,23 @@ public override IdentifierNode GetAstIdentifierForOutputIndex(int outputIndex)
             }
         }
 
+        /// <summary>
+        /// The method returns the assembly name from which the node originated.
+        /// </summary>
+        /// <returns>Assembly Name</returns>
+        internal override AssemblyName GetNameOfAssemblyReferencedByNode()
+        {
+            AssemblyName assemblyName = null;
+
+            var descriptor = this.Controller.Definition;
+            if (descriptor.IsPackageMember)
```

Comment:
I think this would be valid even if this was not a packaged node -right?

Reply:
Yes, it would be, but I think the intent it to return null for non-packaged nodes.

Reply:
then I think your summary/returns could be updated to reflect that / a test should be added encoding that behavior.

Reply:
Sure, just want to add that this was the default behavior, I did not add this code but moved it inside the base class. So whatever it was doing before will still be applied.